platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/PreConsensusEventHandler.java

```diff
@@ -145,13 +138,9 @@ public void clear() {
     }
 
     /**
-     * Do pre-handle processing on the pre-consensus event and add it to the queue (q1) for handling. Events that are
-     * null or empty are discarded immediately. All other events go through pre-handle processing. Events that should
-     * not be handled pre-consensus according to {@link SwirldStateManager#discardPreConsensusEvent(EventImpl)} are not
-     * added to the queue.
+     * Prehandle application transactions, and enqueue platform transactions for prehandling.
```

Comment:
`preHandle()` is guaranteed to occur prior to consensus `handle()`. The statement regarding application transactions is correct. The second part about enqueuing platform transactions for prehandling is not correct because that handle method is not guaranteed to occur before consensus handle. That is why it is referred to with a different term: preConsensus handle.

The naming here is not great, i admit. But let's not use the same term for two different things.

Reply:
Reverted some of this javadoc. I've removed parts that are no longer relevant.